intl/fr/messages.json

```diff
@@ -0,0 +1,31 @@
+{
+    "all1": "",
```

Comment:
that's one of the GPB issues.  For es, fr, and pt, this string is translated to empty most likely due to the double curly braces.  The issue has been reported to the GPB team.